### PR TITLE
Storing theme (light/dark) in localStorage

### PR DESCRIPTION
**Pull request by PhysicistSouravDas**
I have added a _feature_ by which if a user selects **dark-mode**, and then *refreshes* the page (or visits the site *later*), then the website will be shown to the user in **dark-mode**. The *theme* is saved in the browser of the user.

I have added the corresponding **.js** and **.min.js** files. 

There is only one thing more to do. The toggling switch is not preserving its previous state. I tried to modify it but I can't. And I do not have any experience with *semantic-ui*.

### CODE REVIEW

**Comment by mrouhi13** [comment]
Thank you for contributing and adding this feature. I found an example in Semantic UI documentation.

https://semantic-ui.com/modules/checkbox.html#/examples

File: static/js/main.js [hunk 1]
```diff
@@ -64,14 +65,21 @@ ethereumCp.on('success', function (e) {
 });
 
 $('h1.ui.header').transition('pulse');
+if (theme && theme === 'dark')  {
+    $('.light').toggleClass('dark').removeClass('light');
+    $('#github-logo').attr('src', 'static/images/gh-mark-light.svg');
+    $('div#dark-mode').addClass('checked');
```
Comment (mrouhi13) [inline static/js/main.js:71]: According to Semantic UI docs, I think this line must be replaced with this:
```
$('#dark-mode.checkbox').checkbox('attach events', '.check.button', 'check');
```

**Review by mrouhi13 — CHANGES_REQUESTED** [review]
https://github.com/mrouhi13/djecrety/pull/20#discussion_r499161178

File: static/js/main.js (hunk 1 above)
Comment (PhysicistSouravDas) [reply static/js/main.js:71]: Well, I tested out this, and this too was not working. But the below line is working nicely (and I got this idea from the Semantic UI docs and Developer Tools)
```javascript
$('div#dark-mode input').attr('checked', 'checked');
```
I am updating the corresponding .js and .min.js files and will update this pull request soon.

**Review by PhysicistSouravDas — COMMENTED** [review]
Fixed the bug with the slider. Now slider **preserves** its previous state (along with the theme) if the webpage is refreshed.

**Review by mrouhi13 — APPROVED** [review]

**Comment by mrouhi13** [comment]
Nice! 👍🏼

**Pull request merged**